gradle/dependencies.gradle

```diff
@@ -21,7 +21,9 @@ ext {
       coveralls  : '2.8.1',
       spotbugs   : '1.3',
       sonarqube  : '2.5',
-      buildConfig: '1.1.8'
+      buildConfig: '1.1.8',
+      dependencyGraph: '0.3.0'
```

Comment:
There's a new version available 👀 `0.4.0` https://github.com/vanniktech/gradle-dependency-graph-generator-plugin/releases/tag/0.4.0 We should bump it and use the most up to date.